src/cplscheme/MultiCouplingScheme.hpp

```diff
@@ -45,7 +46,8 @@ class MultiCouplingScheme : public BaseCouplingScheme {
       std::map<std::string, m2n::PtrM2N> m2ns,
       constants::TimesteppingMethod      dtMethod,
       const std::string &                controller,
-      int                                maxIterations = -1);
+      int                                maxIterations      = -1,
+      int                                extrapolationOrder = -1);
```

Comment:
MultiCouplingScheme only exists in implicit mode, right?
One could argue that there should be no default value here then.

Reply:
Good point. Did this in 4f2a110715aa948b4240107445232c4d8c83b32b.